source/features/linkify-labels-on-dashboard.tsx

```diff
@@ -8,9 +8,13 @@ import {wrap} from '../helpers/dom-utils';
 import features from '.';
 
 function init(): void {
-	observe('.js-recent-activity-container :not(a) > .IssueLabel', {
+	const labelClass = [
+		'.js-recent-activity-container :not(a) > .IssueLabel', // Recent activity
+		'.js-all-activity-header + div :not(a) > .IssueLabel' // Newsfeed
+	].join();
+	observe(labelClass, {
 		add(label) {
-			const activity = label.closest('li')!;
+			const activity = label.closest('li')! ?? label.parentElement; // Newsfeed
```

Comment:
Does this have to go in its own PR?

Reply:
Yes. It's not correct now because `isPR` will never match.

Reply:
What?

Reply:
```js
const isPR = select.exists('.octicon-git-pull-request', activity);
```

There's never a `octicon-git-pull-request` inside `label.parentElement`. Just drop this change from this PR for now.

Reply:
I will I forgot to mention it only is there if `hide-usless-feeds` is disabled. Regardless I am reverting.

Reply:
Indeed. Let's not add that at all